packages/xo-server/package.json

```
@@ -29,7 +29,7 @@
     "bin": "bin"
   },
   "engines": {
-    "node": ">=10"
+    "node": ">=11.15"
```

https://node.green UI is not intuitive but the first Node version supporting `Array.flatMap` is 11.0.0

```suggestion
    "node": ">=11"
```